components/PaginationLinks.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Link from "next/link";
+
+const PaginationLinks = ({offset, username}) => {
+  const offset2 = offset || 0;
+  let olderLink = '/' + username + '?offset=' + (+offset2 + 30);
+  let newerLink = '/' + username + '?offset=' + (+offset2 - 30);
+
+  return <>
+    <ul className="pagination">
+      <li className="newer">
+        { offset2 > 0 &&
+        <Link href={newerLink}>
+          ← Newer entries
+        </Link>
+        }
+      </li>{' '}
+
+      <li className="older">
+        <Link href={olderLink}>
+          Older entries →
+        </Link>
+      </li>
+    </ul>
+
+    <style jsx>{`
+      .pagination {
```

Comment:
I think we can just select `ul` here, since it's the only `ul` in the component, and get rid of the classname.